tools/integration-tests/tester/tests/consensus/consensus_conflict_spam_test.go

```diff
@@ -0,0 +1,185 @@
+package consensus
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/iotaledger/hive.go/crypto/ed25519"
+
+	"github.com/iotaledger/goshimmer/packages/consensus/gof"
+	"github.com/iotaledger/goshimmer/packages/jsonmodels"
+	"github.com/iotaledger/goshimmer/packages/ledgerstate"
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/framework"
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/tests"
+	"github.com/stretchr/testify/require"
+)
+
+// constant var, shouldn't be changed
+var tokensPerRequest int
+
+/**
+TestConflictSpam spams a node with conflicts and makes sure the GoFs are the same across the network
+*/
+func TestConflictSpam(t *testing.T) {
+	ctx, cancel := tests.Context(context.Background(), t)
+	defer cancel()
+	n, err := f.CreateNetwork(ctx, t.Name(), 4, framework.CreateNetworkConfig{
+		Faucet:      true,
+		StartSynced: true,
+		Activity:    true,
+	}, tests.EqualDefaultConfigFunc(t, false))
+	require.NoError(t, err)
+	defer tests.ShutdownNetwork(ctx, t, n)
+
+	faucet, peer1 := n.Peers()[0], n.Peers()[1]
+	tokensPerRequest = faucet.Config().TokensPerRequest
+
+	tests.AwaitInitialFaucetOutputsPrepared(t, faucet, n.Peers())
+
+	fundingAddress := peer1.Address(0)
+	tests.SendFaucetRequest(t, peer1, fundingAddress)
+	require.Eventually(t, func() bool {
+		return tests.Balance(t, peer1, fundingAddress, ledgerstate.ColorIOTA) >= uint64(tokensPerRequest)
+	}, tests.Timeout, tests.Tick)
+
+	addresses := make([]*ledgerstate.Address, 100)
+	keyPairs := map[string]*ed25519.KeyPair{}
+	for i := 0; i < 100; i++ {
+		address := peer1.Address(i)
+		addresses[i] = &address
+		keyPairs[address.String()] = peer1.KeyPair(uint64(i))
+	}
+
+	outputs := getOutputsControlledBy(t, peer1, fundingAddress)
+	fundingKeyPair := map[string]*ed25519.KeyPair{fundingAddress.String(): peer1.KeyPair(0)}
+	outputs = splitToAddresses(t, peer1, outputs[0], fundingKeyPair, addresses...)
+	const conflictRepetitions = 4
+	pairwiseOutputs := outputs[:(conflictRepetitions-1)*3+3]
+	tripletOutputs := outputs[len(pairwiseOutputs):]
+
+	txs := []*ledgerstate.Transaction{}
+	for i := 0; i < conflictRepetitions; i++ {
+		sendPairWiseConflicts(t, n.Peers(), pairwiseOutputs[i*3:i*3+3], keyPairs, &txs, i)
+		sendTripleConflicts(t, n.Peers(), tripletOutputs[i*3:i*3+3], keyPairs, &txs, i)
```

Comment:
`&txs` doesn't look right. A slice is already a pointer.

Reply:
Not exactly. A slice consists of a pointer to an array and a header that contains the length and capacity of the array.  
Without `&` I would be copying the pointer and the header. Then the append operation I am doing to the slice would update the header copy but not the original slice.

This is according to https://go.dev/blog/slices-intro

Reply:
That's right. Though it's not very readable imo. Maybe better to return the created txs in a slice and just append the returned slice. Something like this:
```go
tx = append(tx, sendTripleConflicts(t, n.Peers(), tripletOutputs[i*3:i*3+3], keyPairs, i)...)
```